plugins/toolbox/src/components/Root/ToolPage.tsx

```diff
@@ -32,8 +35,14 @@ export const ToolPage = (props: ToolsPageProps) => {
         }
       >
         <ContentHeader
-          title={`${tool.category} - ${tool.name}`}
-          description={tool.description}
+          title={`${tool.category} - ${i18n_UNSAFE(
```

Comment:
Maybe try if the Multiple fallback keys works here; https://www.i18next.com/translation-function/essentials#multiple-fallback-keys

Reply:
Or `t(`tool.${tool.id}.name`, {defaultValue: tool.name})` in options? Not sure, should try this out 😅